src/site/filters/liquid.js

```diff
@@ -8,10 +8,7 @@ function getPath(obj) {
 }
 
 module.exports = function registerFilters() {
-  const {
-    featureFlags,
-    enabledFeatureFlags,
-  } = require('../utilities/featureFlags');
+  const { cmsFeatureFlags } = global;
```

Comment:
Because we'll be working with other kinds of feature flags later, I decided to rename this from `enabledFeatureFlags` to `cmsFeatureFlags` to be specific.